deps-s390x.txt

```diff
@@ -0,0 +1 @@
+src/deps-s390x.txt
```

Comment:
I'm fine with this since there are already these symlinks, but why do they exist in the first place? Thoughts on removing them and just getting the whole codebase to use `src/deps-*` in a different PR?

Or we have enough we could even move them to their own directory, given that they aren't actually source.